packages/sdk-browser/scripts/codegen.js

```diff
@@ -469,22 +469,66 @@ export class TurnkeySDKClientBase {
   }`,
       );
     }
+
     // generate a stamping method for each method
-    codeBuffer.push(
-      `\n\tstamp${operationNameWithoutNamespace} = async (input: SdkApiTypes.${inputType}): Promise<TSignedRequest | undefined> => {
+    if (methodType === "noop") {
+      // we skip stamp method generation for noop methods
+      continue;
+    } else if (methodType === "query") {
+      // for query methods, we use flat body structure
+      codeBuffer.push(
+        `\n\tstamp${operationNameWithoutNamespace} = async (input: SdkApiTypes.${inputType}): Promise<TSignedRequest | undefined> => {
     if (!this.stamper) {
       return undefined;
     }
+
+    let session = await getStorageValue(StorageKeys.Session);
+    session = parseSession(session!);
     const fullUrl = this.config.apiBaseUrl + "${endpointPath}";
-    const body = JSON.stringify(input);
-    const stamp = await this.stamper.stamp(body);
+    const body = {
+      ...input,
+      organizationId: input.organizationId ?? session?.organizationId ?? this.config.organizationId
+    };
+
+    const stringifiedBody = JSON.stringify(body);
+    const stamp = await this.stamper.stamp(stringifiedBody);
     return {
-      body: body,
+      body: stringifiedBody,
       stamp: stamp,
       url: fullUrl,
     };
   }`,
-    );
+      );
+    } else {
+      // for command and activityDecision methods, both use the same stamp structure
+      codeBuffer.push(
+        `\n\tstamp${operationNameWithoutNamespace} = async (input: SdkApiTypes.${inputType}): Promise<TSignedRequest | undefined> => {
+    if (!this.stamper) {
+      return undefined;
+    }
+
+    const { organizationId, timestampMs, ...parameters } = input;
+    let session = await getStorageValue(StorageKeys.Session);
+    session = parseSession(session!);
+
+    const fullUrl = this.config.apiBaseUrl + "${endpointPath}";
+    const bodyWithType = {
+      parameters,
+      organizationId: organizationId ?? (session?.organizationId ?? this.config.organizationId),
```

Comment:
if we're strictly enforcing `session` to be non-null above (via `session!`), seems like there's no need for the `?` here